perfeed/settings/configs.toml

```diff
@@ -1,6 +1,6 @@
 [config]
 openai_model="gpt-4o-mini"
-ollama_model="llama3.1:8b"
```

Comment:
if u install llama3.1 on ollama now, it defaults 8b. gonna follow their naming convention